src/main/resources/openapi.json

```diff
@@ -1104,13 +1104,60 @@
     },
     "components": {
         "schemas": {
+            "KafkaHeader": {
+                "example": {
+                    "key": "key1",
+                    "value": "value1"
+                },
+                "properties": {
+                    "key": {
+                        "type": "string"
+                    },
+                    "value": {
+                        "type": "string"
```

Comment:
I would like to start a discussion around this.
As for the Kafka protocol, the header value is just an array of bytes so it could contain binary data as well and not just string. Did you think about that? Is there any specific reason you decided to support string only? for your use case?
As for the value message body, the current bridge support JSON and binary encoding which is specified in the `format` field when a consumer is created. Should we provide support for a more generic possible way to encode header value as well? Just thinking aloud ..... @stanlyDoge @scholzj wdyt?

Reply:
String was enough for my use case, but indeed it might not be generic enough.
I can add a `format` field (default should probably be` String` ?) at `KafkaHeader` level that would qualify the format of the header value.

```
"kafkaRecord": {
  "key": "key1",
  "partition": 0,
  "value": "value1",
  "headers": [
    {
      "key": "key1",
      "value": "string default format"
    },
    {
      "key": "key2",
      "format": "string"
      "value": "string format"
    },
    {
      "key": "key3",
      "format": "binary"
      "value": "<base 64 encoded bytes>"
    }
  ]
}
```

Reply:
Yes, I guess I got your point  : sticking to Kafka's wire format for headers  - that would be raw binary - as  the official Kafka APIs do not provided facilities to serialize / deserialize Headers in the first place. 
In other words you'd like the Strimzi HTTP bridge to provide the same level of service as the official APIs for what concerns headers.

While this makes perfectly sense, in real life headers will be mostly used to express messaging metadata and I assume that this would be commonly done in the form of key/value String pairs; an hypothetical base64 encoding / decoding overhead is minimal but I guess it would be handy to benefit from human readable headers within the bridge interfaces to further ease investigations and testing.

Reply:
@codeJack now that we agreed on the base64 encoded value, we should document it.
I think that in the "value" and alongside "type", the OpenAPI spec should support a "description" to make it clear. Wdyt?